wcfsetup/install/files/lib/system/template/plugin/CsrfTokenFunctionTemplatePlugin.class.php

```diff
@@ -0,0 +1,38 @@
+<?php
+// @codingStandardsIgnoreFile
+namespace wcf\system\template\plugin;
+use wcf\system\exception\SystemException;
+use wcf\system\template\TemplateEngine;
+
+/**
+ * Template compiler plugin that prints the CSRF token ("Security Token").
+ * 
+ * Usage:
+ * 	{csrfToken}
+ * 	{csrfToken type=raw}
+ * 	{csrfToken type=url}
+ * 
+ * @author	Tim Duesterhus
+ * @copyright	2001-2020 WoltLab GmbH
+ * @license	GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package	WoltLabSuite\Core\System\Template\Plugin
+ */
+class CsrfTokenFunctionTemplatePlugin implements IFunctionTemplatePlugin {
+	/**
+	 * @inheritDoc
+	 */
+	public function execute($tagArgs, TemplateEngine $tplObj) {
+		if (isset($tagArgs['type']) && $tagArgs['type'] === 'raw') {
+			return \wcf\system\WCF::getSession()->getSecurityToken();
```

Comment:
Why are you not importing the class? (also below)

Reply:
Historic reasons. I've initially used a BlockTemplatePlugin which allowed me to directly emit PHP code, but it came with a few drawbacks. That's why I opted for Function + Prefilter and didn't adjust this.

I'll fix this in session-meta and then update this PR.